src/components/20-molecules/footer-small/index.scss

```diff
@@ -21,13 +19,32 @@
     justify-content: space-between;
   }
 
-  &__side-aligned {
-    padding-top: 5px;
-    padding-bottom: 5px;
+  span {
```

Comment:
no globals. ever

Reply:
I always use those for testing and then forget to clean up... thx.

Reply:
and i will always be there and checking ahahahaha